website/cue/reference/components/aws.cue

```diff
@@ -39,6 +39,16 @@ components: _aws: {
 							examples: ["arn:aws:iam::123456789098:role/my_role"]
 						}
 					}
+					region: {
+						category:    "Auth",
+						common: 	 false
+						description: "The [AWS region](\(urls.aws_regions)) to call STS against"
```

Comment:
```suggestion
						description: "The [AWS region](\(urls.aws_regions)) to send STS requests to. If not set, this will default to the configured region for the service itself."
```